components/button/README.md

```diff
@@ -69,68 +81,32 @@ To make your `d2l-button-subtle` accessible, use the following properties when a
 | `aria-haspopup` | [Indicate clicking the button opens a menu](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-haspopup). Example: [d2l-dropdown](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/dropdown-opener-mixin.js#L46). |
 | `description` | Use when text on button does not provide enough context. |
 
-## d2l-button-icon
+## Icon Button [d2l-button-icon]
 
 The `d2l-button-icon` element can be used just like the native `button`, for instances where only an icon is displayed.
 
-![Icon Button](./screenshots/button-icon.png?raw=true)
-
 ```html
+<!-- docs: live demo
+name: d2l-button-icon
+defaults: { "text": "My Button", "icon": "tier1:gear" }
+-->
 <script type="module">
   import '@brightspace-ui/core/components/button/button-icon.js';
 </script>
-<d2l-button-icon text="My Button" icon="tier1:gear"></d2l-button-icon>
+<d2l-button-icon $attributes></d2l-button-icon>
 ```
 
-**Properties:**
-
-| Property | Type | Description |
-|--|--|--|
-| `icon` | String, required | [Preset icon key](../icons#preset-icons) (e.g. `tier1:gear`) |
-| `text` | String, required | Accessible text for the buton |
-| `disabled` | Boolean | Disables the button |
-| `disabledTooltip` | String | Tooltip text when disabled |
-| `h-align` | String | Possible values are undefined (default) or `text`. If `text`, aligns the button content to the leading edge of text. |
-| `translucent` | Boolean | Indicates to display translucent (ex. on rich backgrounds) |
-
-**Accessibility:**
+### Accessibility Properties
 
 To make your `d2l-button-icon` accessible, use the following properties when applicable:
 
 | Attribute | Description |
 |--|--|
+| `text` | **REQUIRED**. Acts as a primary label and tooltip. |
 | `aria-expanded` | [Indicate expansion state of a collapsible element](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-expanded). Example: [d2l-more-less](https://github.com/BrightspaceUI/core/blob/f9f30d0975ee5a8479263a84541fc3b781e8830f/components/more-less/more-less.js#L158). |
 | `aria-haspopup` | [Indicate clicking the button opens a menu](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-haspopup). Example: [d2l-dropdown](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/dropdown-opener-mixin.js#L46). |
-| `text` | Acts as a primary label and tooltip and is **REQUIRED**. |
 | `aria-label` | Acts as a primary label. If `text` AND `aria-label` are provided, `aria-label` is used as the primary label, `text` is used as the tooltip. |
 
-## d2l-floating-buttons
```

Comment:
Floating buttons have been moved into a separate document but have not been prepped as a Daylight site page.